src/service/AppealStorageService.ts

```diff
@@ -28,12 +29,14 @@ export class AppealStorageService {
 
         const uri: string = `${this.uri}/companies/${appeal.penaltyIdentifier.companyNumber}/appeals`;
 
-        console.log('Making a POST request to ' + uri);
+        loggerInstance()
+            .debug('Making a POST request to ' + uri);
 
         return await axios
             .post(uri, appeal, config)
             .then((response: AxiosResponse) => {
                 if (response.status === CREATED && response.headers.location) {
+                    loggerInstance().info(`${AppealStorageService.name} - save: Created appeal`);
```

Comment:
This is not as useful as it could be. We could log:
- company number and link to resource created in Appeals API or
- link where POST request was made and link to resource created in Appeals API (to make it 'symmetric')

E.g.

```
Making a POST request to ${uri}

POST request to ${uri} created resource ${response.headers.location}
```